source/config/__init__.py

```diff
@@ -1,3 +1,9 @@
+#config\__init__.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2006-2016 NV Access Limited

```

Comment:
Thanks for adding the copyright. It should be:

```
#Copyright (C) 2006-2016 NV Access Limited, Aleksey Sadovoy, Peter Vágner, Rui Batista, Zahari Yurukov
```

Note that this will also require you to add an encoding declaration to the top of the file and save as UTF-8 without BOM. If this causes a heap of trouble, never mind; I can deal with it later.